### PR TITLE
Changed href of login link on donation form

### PR DESCRIPTION
**Pull request by parthnagpal17**
Changed the href on the login links in the donation form @ML-Chen @KoolCards

### CODE REVIEW

File: components/donation/form/DonationPageFormAdminLoginLink.tsx [hunk 1]
```diff
@@ -6,6 +6,8 @@ import Typography from "@material-ui/core/Typography";
 
 import HorizonLink from "@horizon/HorizonLink";
 
+import Config from "../../../config";
```
Comment (PC98) [inline components/donation/form/DonationPageFormAdminLoginLink.tsx:9]: just do `import config from "config"` please, we have set-up module aliasing 🙂

Comment (parthnagpal17) [reply components/donation/form/DonationPageFormAdminLoginLink.tsx]: Just fixed it!

Comment (PC98) [reply components/donation/form/DonationPageFormAdminLoginLink.tsx]: I think you forgot to fix the path 😅
The idea is that using relative paths like `from "../../../config";` is not a good idea/makes code harder to maintain and refactor.
That's why, in our repo, we let you do `from "config"` whenever importing the config object.

Comment (parthnagpal17) [reply components/donation/form/DonationPageFormAdminLoginLink.tsx]: Oh, my bad. I got it now, thanks :)

**Review by PC98 — APPROVED** [review]

**Pull request merged**